src/ripple/protocol/Seed.h

```diff
@@ -0,0 +1,35 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2015 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_PROTOCOL_SEED_H_INCLUDED
+#define RIPPLE_PROTOCOL_SEED_H_INCLUDED
+
+#include <ripple/basics/base_uint.h>
+#include <ripple/json/json_value.h>
+#include <ripple/protocol/RippleAddress.h>
+
+namespace ripple {
+
+uint256 keyFromSeed (uint128 const& seed);
+
+RippleAddress getSeedFromRPC (Json::Value const& params);
```

Comment:
I think we could just stuff these declarations in RippleAddress.h, no need for an extra pair of files. Also, are these really protocol-level features? Or are they just utility functions? Does any part of the protocol depend on this code (I don't think converting JSON to RippleAddress is not part of the protocol, might be wrong though).